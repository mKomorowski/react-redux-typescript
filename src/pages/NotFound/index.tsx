import React, { FC } from "react";
import { Link } from "react-router-dom";

const NotFound: FC = () => (
  <div className="not-found">
    <h2>Page not found</h2>
    <Link to="/">Return to the Home Page</Link>
  </div>
);

export default NotFound;
