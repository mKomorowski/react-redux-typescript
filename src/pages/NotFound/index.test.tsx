import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";

import NotFound from "./index";

describe("NotFound component", () => {
  it("renders correctly", () => {
    const { container } = render(
      <Router>
        <NotFound />
      </Router>,
    );

    expect(container).toMatchSnapshot();
  });
});
