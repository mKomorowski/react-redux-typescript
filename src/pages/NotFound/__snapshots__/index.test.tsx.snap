// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component renders correctly 1`] = `
<div>
  <div
    class="not-found"
  >
    <h2>
      Page not found
    </h2>
    <a
      href="/"
    >
      Return to the Home Page
    </a>
  </div>
</div>
`;
